### ISSUE TITLE
Build fails with `ENAMETOOLONG: name too long, open` error

### ISSUE THREAD

**Issue by dgolovin** [openshift/console#9149]
When I am running build on Ubuntu 21.04 from master branch and getting the error below. I am following Quickstart readme section https://github.com/openshift/console#quickstart.
```
$ ./build.sh
[webpack-cli] [Error: ENAMETOOLONG: name too long, open '/home/denis/p/console/frontend/public/dist/vendors~admin-pipeline~admin-tasks`~admin-triggers~delete-revision~dev-console-add~dev-console-deplo~01c3391d-chunk-542127130c88b772b998.min.js.map'] {
  errno: -36,
  code: 'ENAMETOOLONG',
  syscall: 'open',
  path: '/home/denis/p/console/frontend/public/dist/vendors~admin-pipeline~admin-tasks`~admin-triggers~delete-revision~dev-console-add~dev-console-deplo~01c3391d-chunk-542127130c88b772b998.min.js.map'
}
```
Is there any way around this?
Thanks!

**Comment by spadgett** [comment]
It looks like this is the same as https://github.com/webpack/webpack/issues/12883 and we need to update to webpack 5 for a fix. Unfortunately, I'm not sure all of the loaders we use support webpack 5.

@vojtechszocs FYI

**Comment by spadgett** [comment]
Story to update to webpack 5: https://issues.redhat.com/browse/CONSOLE-2971

/remove-lifecycle stale

**Comment by spadgett** [comment]
/remove-lifecycle rotten

This should be fixed by #10204
